Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml

```diff
@@ -8249,6 +8279,7 @@
     </CModel>
     <CModel id="AP_ScourgePortrait" parent="Portrait" Race="Zerg">
         <Model value="Assets\portraits\zerg\scourgeportrait\scourgeportrait.m3"/>
+        <Image value="AP\Assets\Textures\ScourgePortraitStatic.dds"/>

```

Comment:
Checking the game files, I couldn't find a static scourge portrait, so looks like adding one here is necessary. However, the image may be off because I also noticed we're missing a lightdata entry for `AP_ScourgePortrait`. We're also missing the entries for ravagers and corruptors, and I only noticed those as they're the ones I remember adding.

Reply:
I added a light entry for the scourge portrait, you were right about that one not existing. The ravager and corruptor do not need light entries, however; they use light entries that already exist in the liberty.sc2mod files. The light entries don't matter for static images since it has no effect on them.